environment-windows.yml

```diff
@@ -3,14 +3,21 @@ channels:
 dependencies:
   - clang-tools
   - conda=4.6
+  - doxygen
```

Comment:
If you could pin these to specific versions, it will:
* make conda run much faster since it doesn't have to consider many options when trying to solve deps
* ensure reproducible builds in the future and for new clones